### DISCUSSION TITLE
Constant WP_REDIS_PREFIX is not found

### DISCUSSION THREAD

**Discussion by BNBox** [rhubarbgroup/.github#54] [Support (ROC)] [answered]
Hello,
Some of our sites have WP_CACHE_KEY_SALT on wp-config.php which I know has been renamed by WP_REDIS_PREFIX in version 2.0, but I still don't understand why some websites do not have any of these constants on their wp-config file.

How are these constants generated automatically?

I understand that I can add them manually, and this is fine, but is there a way they can be automatically generated?

Thank you

**Comment by tillkruss** [comment, marked as answer]
Redis Object Cache does not generate these constants, must be your hosting provider.

They are not needed if you give each site their own `database` in on your Redis Server. It's a much cleaner approach.
